components/automate-ui/src/app/entities/api-tokens/api-token.reducer.ts

```diff
@@ -45,7 +45,12 @@ export function apiTokenEntityReducer(
   switch (action.type) {
 
     case ApiTokenActionTypes.GET_ALL: {
-      return set(STATUS, EntityStatus.loading, state) as ApiTokenEntityState;
+      return set(
+        STATUS,
+        EntityStatus.loading,
+        // clear token state to avoid leaking potentially unauthorized data
```

Comment:
I think these comments are a bit misleading. We are only showing data the user is authorized to see via the API. We are working around a quark in our API permissions, not leaking any new data.